### ISSUE TITLE
Communication between micro apps

### ISSUE THREAD

**Issue by shuyue426** [bytedance/magic-portal#5]
As we know,each app of magic-portal is a Web Component.
What's the suggest way to communicate between magic-portal apps.

**Comment by h-a-n-a** [comment]
Yes, due to the fact that they are sharing the same global context, and we strictly follow the proposal. It's much like just an `addEventListener` on both side of the global context, we will add more examples to README later.
